### DISCUSSION TITLE
Feature Wish: Diff settings per Sketch file

### DISCUSSION THREAD

**Discussion by getinet** [ingrammicro/puzzle-publisher#79] [General]
First and most, I LOVE LOVE LOVE this plugin. I get more excited with updates to PP plugins over Sketch updates.
I would love to have PP settings that are custom to each sketch file (project) I'm working on.

**Comment by MaxBazarov** [comment]
Thanks for the warm feedback.
Currently I’ll on vacation till 28 July, but will response right after.

**Reply by getinet** [reply]
Well Deserved!
